core/vm/operations_verkle.go

```diff
@@ -138,14 +142,17 @@ func gasExtCodeCopyEIP4762(evm *EVM, contract *Contract, stack *Stack, mem *Memo
 	addr := common.Address(stack.peek().Bytes20())
 
 	if _, isPrecompile := evm.precompile(addr); isPrecompile {
+		var overflow bool
+		if gas, overflow = math.SafeAdd(gas, params.WarmStorageReadCostEIP2929); overflow {
+			return 0, ErrGasUintOverflow
+		}
 		return gas, nil
 	}
 	wgas := evm.Accesses.TouchBasicData(addr[:], false)
 	if wgas == 0 {
 		wgas = params.WarmStorageReadCostEIP2929
 	}
 	var overflow bool
-	// We charge (cold-warm), since 'warm' is already charged as constantGas
```

Comment:
@gballet, if I looked correctly, for the verkle fork there aren't constant gas costs? Or maybe I misinterpreted the comment.

Reply:
yeah that's incorrect. There was a time when it was, but this comment should have been deleted a long time ago.